<launch>

  <node pkg="uvc_camera" name="camera" type="uvc_camera_node" >
    <param name="camera_info_url" type="string" value="file://$(find cugo_teleop)/config/camera.yaml"/>
  </node>

  <node name="image_proc" pkg="image_proc" type="image_proc" />

  <node name="image_raw_view" pkg="image_view" type="image_view" >
    <remap from="image" to="image_raw"/>
  </node>

  <node name="object_detection" pkg="cugo_detection" type="object_detection" output="screen">
    <param name = "hue_mn" type = "int" value="150"/>
    <param name = "hue_mx" type = "int" value="180"/>
    <param name = "sat_mn" type = "int" value="100"/>
    <param name = "sat_mx" type = "int" value="255"/>
    <param name = "value_mn" type = "int" value="50"/>
    <param name = "value_mx" type = "int" value="255"/>
    <param name = "bright_mn" type = "int" value="0"/>
    <param name = "bright_mx" type = "int" value="0"/>
  </node>

</launch>
